aspnetcore/fundamentals/middleware/request-response.md

```diff
@@ -70,6 +70,11 @@ These issues are fixable, but the code is becoming progressively more complicate
 
 ## Pipelines
 
+When writing directly to `HttpResponse.BodyWriter`, call `PipeWriter.FlushAsync()` manually to ensure the data is flushed to the underlying response stream. Here's why:
```

Comment:
```suggestion
When writing directly to `HttpResponse.BodyWriter`, call `PipeWriter.FlushAsync()` manually to ensure the data is flushed to the underlying response data structure. Here's why:
```
Actually it's not a `Stream`. For Kestrel the data is written to socket send more or less directly.
Thus the second bullet below is also a bit misleading.

Reply:
`response body`?